onnxruntime/core/common/threadpool.cc

```diff
@@ -390,7 +393,15 @@ int ThreadPool::DegreeOfParallelism(const concurrency::ThreadPool* tp) {
 #else
   // When not using OpenMP, we parallelise over the N threads created by the pool
   // tp, plus 1 for the thread entering a loop.
-  return tp ? (tp->NumThreads()+1) : 1;
+  if (tp) {
+    if (CPUIDInfo::GetCPUIDInfo().IsHybrid()) {
+      return ((tp->NumThreads() + 1)) * MLAS_GRANULARITY_FACTOR;
```

Comment:
>MLAS_GRANULARITY_FACTOR [](start = 40, length = 23)

Leaks MSLAS library constant into a general purpose code.

Reply:
Not necessarily MLAS_GRANULARITY_FACTOR, will rename to TASK_GRANULARITY_FACTOR